CityOfDetroit/bloom
# 1344/prominent filter button

# Pull Request Template

## Issue Overview

This PR addresses #1344 


## Description

This PR included updates to the Filter button styling on the Detroit Public listings page as well as the content (icon and text). Additionally, the filter button will now drop down a row and expand to full-width on mobile screens.

Update: This ticket implements the font awesome icon on the filtering results page with the recently added FA support. Also, @sheneral said that this PR could move forward without translations since translations are in progress but currently a bit delayed and can be implemented afterwards.

## How Can This Be Tested/Reviewed?

This can be tested on the Detroit public site by going to the listings page and noticing the new "Find Listings for You" button at both desktop and mobile dimensions. Then add some filters and observe the updated icon on the filter button on the results page.
## Checklist:


## Reviewer Notes:

Steps to review a PR:

- Read and understand the issue, and ensure the author has added QA notes
- Review the code itself from a style point of view
- Pull the changes down locally and test that the acceptance criteria is met
- Also review the acceptance criteria on the Netlify deploy preview (noting that these do not yet include any backend changes made in the PR)
- Either explicitly ask a clarifying question, request changes, or approve the PR if there are small remaining changes but the PR is otherwise good to go

## On Merge:

If you have one commit and message, squash. If you need each message to be applied, rebase and merge.
